### ISSUE TITLE
MSVS 16 error impy is not subscriptable

### ISSUE THREAD

**Issue by myhub6997** [hanjinliu/impy#16]
Hi, I have just started out trying to import impy as ip in MSVS 16 2019 with the following statement

import impy as ip

I also tried

from impy import ALL as ip

Both statement fail with VS error that impy is not subscriptable.

Could you please address this issue.

![impy-error](https://user-images.githubusercontent.com/101022539/156901138-85a32ccc-60e4-40b5-8447-9baea5b253d3.png)

My python 3.8 virtual environment is installed under the project repo folders via command line terminal
I have used the command pip install impy-array to install impy in the project's virtual environment.

Thanks for you help

![impy-error-report](https://user-images.githubusercontent.com/101022539/156901492-b51adca9-fb51-48c0-9bc3-22bedaeb98e6.png)

**Comment by hanjinliu** [comment]
Hi @myhub6997 , thank you for reporting the bug!

I was so careless that I did not test the latest version in Python 3.8... I found the problem and fixed it in the latest commit so it will be OK in next `v1.26.1`.
There are still things that I want to edit before releasing `v1.26.1`. If you want to try it now there are three options:

1. Install Python >=3.9 in a new environment (if you don't have `git` this is the only solution).
2. Install the latest commit.
  ```
  pip install git+https://github.com/hanjinliu/impy
  ```
3. Clone the repository.
  ```
  git clone https://github.com/hanjinliu/impy
  ```
